### PR TITLE
ALB for cvatportal.synapse.org

### PR DESCRIPTION
**Pull request by thomasyu888**

### CODE REVIEW

File: config/prod/tom-cvat-alb.yaml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  # The Sage project (Infrastructure, amp-ad, ntap, dream, etc..)
+  Project: "dream"
+  # The resource owner
+  OwnerEmail: "thomas.yu@sagebase.org"
+
+   # load balancer configs
+  Ec2InstanceId: "i-04c318988a159582a"
+  SSLCertificateIdArn: "arn:aws:acm:us-east-1:055273631518:certificate/992c2d49-579a-465e-beff-1a08c284f7db"
+  VpcId: "vpc-0e9b80dc470a797d5"   # computevpc
+  AppPort: "80"         # rstudio port
```
Comment (zaro0508) [inline config/prod/tom-cvat-alb.yaml]: your app is running on port 80? also comment is "rstudio"?

File: config/prod/tom-cvat-alb.yaml [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  Department: "CompOnc"
+  # The Sage project (Infrastructure, amp-ad, ntap, dream, etc..)
+  Project: "dream"
+  # The resource owner
+  OwnerEmail: "thomas.yu@sagebase.org"
+
+   # load balancer configs
+  Ec2InstanceId: "i-04c318988a159582a"
+  SSLCertificateIdArn: "arn:aws:acm:us-east-1:055273631518:certificate/992c2d49-579a-465e-beff-1a08c284f7db"
+  VpcId: "vpc-0e9b80dc470a797d5"   # computevpc
```
Comment (zaro0508) [inline config/prod/tom-cvat-alb.yaml]: vpc id seems correct, however the vpc name is  "sandcastle" not "computevpc". please update comment

File: config/prod/tom-cvat-alb.yaml [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+  # The Sage deparment (Platform, CompOnc, SysBio, Governance, etc..)
+  Department: "CompOnc"
+  # The Sage project (Infrastructure, amp-ad, ntap, dream, etc..)
+  Project: "dream"
+  # The resource owner
+  OwnerEmail: "thomas.yu@sagebase.org"
+
+   # load balancer configs
+  Ec2InstanceId: "i-04c318988a159582a"
+  SSLCertificateIdArn: "arn:aws:acm:us-east-1:055273631518:certificate/992c2d49-579a-465e-beff-1a08c284f7db"
```
Comment (zaro0508) [inline config/prod/tom-cvat-alb.yaml]: the "*.synapse.org" certificate in sandbox is "arn:aws:acm:us-east-1:563295687221:certificate/b7f3a44b-1c72-4f84-8e8a-42050764e6c2"

File: config/prod/tom-cvat-alb.yaml [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+  VpcId: "vpc-0e9b80dc470a797d5"   # computevpc
+  AppPort: "80"         # rstudio port
+  Subnets: "subnet-21a68d7c,subnet-4f3e2b2b,subnet-77d1e458"  # public subnets
```
Comment (zaro0508) [inline config/prod/tom-cvat-alb.yaml]: public subnets are different for sandbox.  please change to:

```
  Subnets: "subnet-02f2acd15e5be53cd, subnet-031359c53fce9b312, subnet-0f19d4c62f9ab0f2c"
```

File: config/prod/tom-cvat-alb.yaml (hunk 1 above)
Comment (thomasyu888) [reply config/prod/tom-cvat-alb.yaml]: Theres two ports. 80 and 8082.

File: config/prod/tom-cvat-alb.yaml (hunk 2 above)
Comment (thomasyu888) [reply config/prod/tom-cvat-alb.yaml]: I dont know what it is.  Please help me figure this out

Comment (zaro0508) [reply config/prod/tom-cvat-alb.yaml]: i just request you to update the comment to "# sandcastlevpc"

File: config/prod/tom-cvat-alb.yaml (hunk 1 above)
Comment (zaro0508) [reply config/prod/tom-cvat-alb.yaml]: hmm, this might be a problem.  will need to take a look at how we might do this.

**Comment by thomasyu888** [comment]
Close because manually linked.

**Pull request closed without merging**